### PR TITLE
Port distance_field to ROS 2

### PR DESCRIPTION
**Pull request by vmayoral**

### CODE REVIEW

**Comment by vmayoral** [comment]
See https://github.com/ros-planning/moveit2/pull/62#issuecomment-480720464

File: moveit_core/distance_field/CMakeLists.txt [hunk 1]
```diff
@@ -7,19 +7,25 @@ add_library(${MOVEIT_LIB_NAME}
...
-install(DIRECTORY include/ DESTINATION ${CATKIN_GLOBAL_INCLUDE_DESTINATION})
-
-if(CATKIN_ENABLE_TESTING)
-  catkin_add_gtest(test_voxel_grid test/test_voxel_grid.cpp)
-  target_link_libraries(test_voxel_grid ${MOVEIT_LIB_NAME})
-
-  catkin_add_gtest(test_distance_field test/test_distance_field.cpp)
-  target_link_libraries(test_distance_field ${MOVEIT_LIB_NAME})
-endif()
+# if(BUILD_TESTING)
```
Comment (mlautman) [inline moveit_core/distance_field/CMakeLists.txt]: Let's get the tests issue figured out before we press ahead with this one https://github.com/AcutronicRobotics/moveit2/issues/13

Comment (davetcoleman) [reply moveit_core/distance_field/CMakeLists.txt]: i believe they have figured out the test issue

Comment (mlautman) [reply moveit_core/distance_field/CMakeLists.txt]: @vmayoral Can we get those changes merged in and cherry picked onto this branch?

**Comment by ahcorde** [comment]
@mlautman, I cherry picked the commits

**Pull request merged**